ingestion/src/metadata/cmd.py

```diff
@@ -246,38 +252,151 @@ def docker(start, stop, clean, type, path) -> None:
                 "Need support? Get in touch on Slack: https://slack.open-metadata.org/",
                 fg="bright_magenta",
             )
-        elif stop:
-            for container in client.containers.list():
-                if "openmetadata_" in container.name:
-                    logger.info(f"Stopping {container.name}")
-                    container.stop()
-        elif clean:
-            logger.info("Removing Containers")
-
-            for container in client.containers.list({all: True}):
-                if "openmetadata_" in container.name:
-                    container.remove(v=True, force=True)
-            logger.info("Removing Networks")
-            for network in client.networks.list():
-                if "app_net" in network.name:
-                    network.remove()
-    except (ImportError, ModuleNotFoundError):
-        click.secho(
-            "Docker package not found, can you try `pip install 'openmetadata-ingestion[docker]'`",
-            fg="yellow",
-        )
+        if pause:
+            logger.info("Pausing docker compose for Open Metadata....")
+            docker.compose.pause()
+            logger.info("Pausing docker compose for Open Metadata Successful.")
+        if resume:
+            logger.info("Resuming docker compose for Open Metadata....")
+            docker.compose.unpause()
+            logger.info("Resuming docker compose for Open Metadata Successful.")
+        if stop:
+            logger.info("Stopping docker compose for Open Metadata....")
+            docker.compose.down(
+                remove_orphans=True, remove_images="local", volumes=True
+            )
+            logger.info("docker compose for Open Metadata stopped successfully.")
+
     except MemoryError:
         click.secho(
             f"Please Allocate More memory to Docker.\nRecommended: 4GB\nCurrent: "
             f"{round(float(docker_info['MemTotal']) / calc_gb, 2)}",
             fg="red",
         )
-    except sys_docker.errors.DockerException as err:
-        click.secho(f"Error: Docker service is not up and running. {err}", fg="red")
     except Exception as err:
         logger.error(traceback.format_exc())
         logger.error(traceback.print_exc())
         click.secho(str(err), fg="red")
 
+    # try:
```

Comment:
lets not keep the commented out code around

Reply:
Done. Thanks Harsha